openpype/hosts/max/api/lib.py

```diff
@@ -314,6 +345,54 @@ def set_timeline(frameStart, frameEnd):
     return rt.animationRange
 
 
+def reset_colorspace():
+    """OCIO Configuration
+    Supports in 3dsMax 2024+
+
+    """
+    if int(get_max_version()) < 2024:
+        return
+    project_name = get_current_project_name()
+    colorspace_mgr = rt.ColorPipelineMgr
+    colorspace_mgr.Mode = rt.Name("OCIO_Custom")
+    ocio_config_path = os.environ.get("OCIO")
+    global_imageio = get_project_settings(
+        project_name)["global"]["imageio"]
+    if global_imageio["activate_global_color_management"]:
+        ocio_config = global_imageio["ocio_config"]
+        ocio_config_path = ocio_config["filepath"][-1]
+
+    max_imageio = get_project_settings(
+        project_name)["max"]["imageio"]
+    if max_imageio["activate_host_color_management"]:
+        ocio_config = max_imageio["ocio_config"]
+        if ocio_config["override_global_config"]:
+            ocio_config_path = ocio_config["filepath"][0]
+            if not ocio_config_path:
+                # use the default ocio config path instead
+                ocio_config_path = ocio_config["filepath"][-1]
+
+    colorspace_mgr.OCIOConfigPath = ocio_config_path
+
+
+def check_colorspace():
+    parent = get_main_window()
+    if parent is None:
+        log.info("Skipping outdated pop-up "
+                 "because Max main window can't be found.")
+    if int(get_max_version()) >= 2024:
+        color_mgr = rt.ColorPipelineMgr
+        if color_mgr.Mode != rt.Name("OCIO_Custom"):
+            from openpype.widgets import popup
+            dialog = popup.Popup(parent=parent)
+            dialog.setWindowTitle("Warning: Wrong OCIO Mode")
+            dialog.setMessage("This scene has wrong OCIO "
+                              "Mode setting.")
+            dialog.setButtonText("Fix")
+            dialog.on_clicked.connect(reset_colorspace)
```

Comment:
```suggestion
            dialog.setStyleSheet(load_stylesheet())
            dialog.on_clicked.connect(reset_colorspace)
```

Requires to add import at the top of the file
```
from openpype.style import load_stylesheet
```